src/harness/fake_sas.py

```diff
@@ -222,6 +237,18 @@ def do_POST(self):
     self.end_headers()
     self.wfile.write(json.dumps(response))
 
+  def do_GET(self):
+    """Handles GET requests."""
+    path, value = self._parseUrl(self.path)
+    if path == "/v1.0/sas_impl":
```

Comment:
SAS-SAS makes a GET Request that's why I have added go_GET method to handle that and then splitting the path into two parts one actual path of the request "v1.0/sas_impl" and another with the ID which is passed in the URL.